vsintegration/src/FSharp.ProjectSystem.FSharp/VSPackage.resx

```diff
@@ -464,10 +464,10 @@
     <value>Customizes the environment to maximize code editor screen space and improve the visibility of F# commands and tool windows.</value>
   </data>
   <data name="ProductDetails" xml:space="preserve">
-    <value>Microsoft Visual F# Tools 10.4 for F# 4.6</value>
```

Comment:
I don't like the hackery in the project file.  I'd rather `ProjectSystem.fsproj` get reverted and these strings updated to: `Microsoft Visual F# Tools {0} for F# {1}`, then call `string.Format()` with the values `FSharp.BuildProperties.fsProductVersion`/`fsLanguageVersion` as generated [here](https://github.com/dotnet/fsharp/blob/3c90593211908287bfe02437bd0bc1c2269ea438/FSharpBuild.Directory.Build.targets#L36-L62).

Reply:
@brettfo,

I feel you … unfortunately these values are read by vs from the resource strings.  We just tell VS the resource IDs it then grabs the resources and displays them.  [Here](https://github.com/dotnet/fsharp/blob/master/vsintegration/Vsix/RegisterFsharpPackage.pkgdef#L20).

We can put the build targets in FSharpBuild.Directory.Build.targets in order to remove the hackery from the project.  The bit I'm still trying to figure out, is how to not break xlf, and yet update the localized versions of  vspackage ... all in all it is quite touchy.

Reply:
@brettfo, Okay fixed it.